### PR TITLE
Added Previous Webinars page

### PR DESCRIPTION
**Pull request by Sc0rpi0n101**
* Created the previous webinars page and moved all webinars to the page
* Changed webinars section to show 3 cards
* changed nav link for webinars to point to the previous webinars page

image

image

image

### CODE REVIEW

File: communities.html [hunk 1]
```diff
@@ -94,7 +94,7 @@
                         <a class="page-scroll" href="index.html#events" data-target="#menuPrimary">Events</a>
                     </li>
                     <li class="nav-item">
-                        <a class="page-scroll" href="index.html#webinars" data-target="#menuPrimary">Webinars</a>
+                        <a href="webinars.html" data-target="#menuPrimary">Webinars</a>
```
Comment (vipulgupta2048) [inline communities.html]:
```suggestion
                        <a href="I want it like that.html" data-target="#menuPrimary">Webinars</a>
```

**Comment by Sc0rpi0n101** [comment]
@vipulgupta2048

**Comment by Sc0rpi0n101** [comment]
> Can I get a reference for the discussion behind this feature?

It was discussed on a call for the redesign, but the issue would be #194 

> Additionally, can you deploy this on a Github Pages or post screenshots for the new page in Mobile version to check how if it behaves responsively (pun)

Alright, on it

**Comment by Sc0rpi0n101** [comment]
It looks fine on mobile too. You can check it out at https://sc0rpi0n101.github.io/asetalias.github.io/

Screenshot_20200228-201256
Screenshot_20200228-201312
Screenshot_20200228-201305

**Comment by thisisayush** [comment]
image

@Sc0rpi0n101 Can you make it align to center

**Comment by Sc0rpi0n101** [comment]
Closing this, for now, would be better to do this with the new design. Might come back later.

**Pull request closed without merging**